lib/Common/Memory/RecyclerWriteBarrierManager.h

```diff
@@ -207,6 +207,8 @@ class RecyclerWriteBarrierManager
     static uint const s_BytesPerCardBit = 1 << s_BitArrayCardTableShift;  // 128 = 1 << 7
     static uint const s_BytesPerCard = s_BytesPerCardBit * 32;      // 4K  = 1 << 12 = 128 << 5
 
+    static Recycler* recycler; // assuming there's only one recycler
```

Comment:
I'm implementing list in Recycler so removing this

---
In reply to: [92254893](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92254893) [](ancestors = 92254893)